content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service:_Deployment_Guide/Back_Up_Key_Material.adoc

```diff
@@ -0,0 +1,8 @@
+== Back Up Key Material
+
+Yubico strongly recommends making a backup copy of all production objects residing on your production devices, particularly once the HGS signing and encryption keys have been generated on the YubiHSM 2. If there is a hardware failure of the production device, having a backup ensures that you can resume operations quickly. The backup process will result in two identical YubiHSM 2 devices with the same number of objects, keys, labels, etc.
+
+Backup the YubiHSM 2 according to the instructions in link:../YubiHSM_2_Windows_Deployment_Guide/Backup_and_Restore_Key_Material.adoc[the corresponding chapter of the YubiHSM 2 Windows Deployment Guide].
```

Comment:
This link doesn't work, the directory is now "YubiHSM_2_Windows_Deployment_Guide--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/"